File: fe_flutter_ui/lib/models/registrationtoken.dart
class RegistrationToken {
  int? id;
  String? deviceToken;
  String? userId;
  bool? status;

  RegistrationToken({this.id, this.deviceToken, this.userId, this.status});

  RegistrationToken.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    deviceToken = json['deviceToken'];
    userId = json['userId'];
    status = json['status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['deviceToken'] = deviceToken;
    data['userId'] = userId;
    data['status'] = status;
    return data;
  }
}

File: be_aspnet_api/MrSoaiAPI/Model/RegistrationToken.cs
using System.ComponentModel;

namespace MrSoaiAPI.Model
{
    public class RegistrationToken
    {
        public int Id { set; get; }
        public string DeviceToken { set; get; }
        public string UserId { set; get; }
        public MrSoaiUser User { set; get; }
        [DefaultValue(true)]
        public bool Status { get; set; } = true;
    }
}
